### PR TITLE
Update youtube-dl

### PR DESCRIPTION
**Pull request by TheRealKeto**
This PR updates ``youtube-dl`` to version 2021.05.16, while adding the Homepage field to the package's control file.

### CODE REVIEW

**Comment by Diatrus** [comment]
Holding off on merging until we merge pandoc to Procursus.

**Review by marruffohomero — APPROVED** [review]

**Pull request merged**
